modules/microshift-gitops-debug.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-debug_{context}"]
+= Debugging GitOps
+
+There are two ways to debug GitOps in your {microshift-short} instance: using an sos report and using the {oc-first}.
+
+.Prerequisites
+
+* The `sos` package is installed.
+* The `oc` command line tool is installed.
+
+.Procedure
+
+. Using sos reports to debug GitOps.
+
+.. Log in to the host as a root user.
+.. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+microshift-sos-report
```

Comment:
In some environments, this command will fail because `microshift-sos-report` is missing. In that case you can get it from here: https://github.com/openshift/microshift/blob/main/scripts/microshift-sos-report.sh Not sure if this should be a prerequisite

Reply:
Ah, I see. I restructured a bit to include the module we already have on running an sos report. Better!